xlsxwriter/xmlwriter.py

```diff
@@ -52,10 +52,8 @@ def _xml_declaration(self):
 
     def _xml_start_tag(self, tag, attributes=[]):
         # Write an XML start tag with optional attributes.
-        for key, value in attributes:
```

Comment:
The changes to xmlwriter should be in a separate commit.

Reply:
do you want me to separate the commit or can I leave as-is?  I don't think i've interactively rebased and divided a commit before, but I'm guessing it's possible.

Reply:
No. Just omit the change as below.